packages/sanity/src/core/components/index.ts

```diff
@@ -21,6 +21,5 @@ export * from './textWithTone'
 export * from './TooltipOfDisabled'
 export * from './transitional'
 export * from './userAvatar'
-export * from './validation'
```

Comment:
Perhaps silly question, but since we are exporting it is there a risk that this would then be a breaking change? 🤔

Reply:
Seems like the scheduled publishing plugin was using this: 

https://github.com/sanity-io/sanity-plugin-scheduled-publishing/issues/80 😬

Reply:
I guess this is as good time as any to understand what is our process / stance when internally tagged tools are used and then we do changes / remove them 😬